docs/going-into-production.rst

```diff
@@ -0,0 +1,357 @@
+``node.data`` set to ``true``.
+
+
+.. _prod-node-d:
+
+Request handling and query execution nodes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure a node that only handles client requests and query execution
+(i.e., is not master-eligible) like this:
+
+.. code-block:: yaml
+
+    node.master: false
+    node.data: true
+
+A good name for this node might be:
+
+.. code-block:: yaml
+
+    node.name: node-02-d
+
+Here, ``node`` is a base name with a sequence number of ``02``. Every node in
+the cluster should have a unique sequence number, independent of the node type.
+The letter ``d`` indicates that this node has ``node.data`` set to ``true``.
+
+
+.. _prod-node-m:
+
+Cluster management nodes
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure a node that handles cluster management (i.e., is
+master-eligible) but does not handle query execution loads like this:
+
+.. code-block:: yaml
+
+    node.master: true
+    node.data: false
+
+A good name for this node might be:
+
+.. code-block:: yaml
+
+    node.name: node-03-m
+
+Here, ``node`` is a base name with a sequence number of ``03``. Every node in
+the cluster should have a unique sequence number, independent of the node type.
+The letter ``m`` indicates that this node has ``node.master`` set to ``true``.
+
+
+.. _prod-node:
+
+Request handling nodes
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure a node that handles client requests but does not handle query
+execution loads or cluster management (i.e., is not master-eligible) like this:
+
+.. code-block:: yaml
+
+    node.master: false
+    node.data: false
+
+A good name for this node might be:
+
+.. code-block:: yaml
+
+    node.name: node-04
+
+Here, ``node`` is a base name with a sequence number of ``04``. Every node in
+the cluster should have a unique sequence number, independent of the node type.
+The absense of any additional letters indicate that ``node.master`` and
+``node.data`` are set to ``false``.
+
+
+.. _prod-config-hostname:
+
+Bind nodes to logical hostnames
+===============================
+
+By default, CrateDB binds to the loopback address (i.e., `localhost`_). It
+listens on port 4200-4299 for HTTP traffic and port 4300-4399 for node-to-node
+communication. Because CrateDB uses a port range, if one port is busy, it will
+automatically try the next port.
+
+When using :ref:`multiple hosts <multi_node_setup>`, nodes must bind to a
+non-loopback address.
+
+.. CAUTION::
+
+      Never expose an unprotected CrateDB node to the public internet
+
+You can bind to a non-loopback address with the `network.host`_ setting in your
+`configuration`_ file, like so:
+
+.. code-block:: yaml
+
+    network.host: node-01-md.acme-prod.internal.example.com
+
+You can break down that hostname, as follows:
+
+- ``example.com`` -- Your example domain name
+- ``internal`` -- Your internal private network
+- ``acme-prod`` -- The cluster name
+- ``node-01-md`` -- The node name
```

Comment:
The DNS can't be configured here, this is given and defined somewhere else. 
To me this reads as if I have the option here.

The break down doesn't really provide any value, because this could be `anything`. Of course we can recommend how to do it, but then we should make it clear. As said, I'd even then doubt the value.

Reply:
I have clarified the point about setting this in DNS

I have also clarified the point that you should use the hostname to logically describe the node

as for the value of doing this: I added this stuff because in your previous review you requested that I provide suggestions re how to name stuff